src/cr/cube/matrix/cubemeasure.py

```diff
@@ -268,6 +298,42 @@ def table_bases(self):
             "`%s` must implement `.table_bases`" % type(self).__name__
         )
 
+    @lazyproperty
+    def _columns_pruning_base(self):
+        """1D bool np.ndarray of the sum of the column-bases for cells in a column
+
+        Used to calculate he columns-pruning-mask, which indicates when every cell in a
```

Comment:
s/he/the

This show up copy/pasted a bunch of times below.

Also s/overriden/overridden in the following paragraph, also repeated several times.